### PR TITLE
🌱 Bumping CAPI to 1.5.2

### PR DESCRIPTION
**Pull request by adilGhaffarDev**
**What this PR does / why we need it**:
- Bump CAPI to v1.5.2
- Bump Cert-manager to v1.13.0
- Bump GO to 1.20.8

### CODE REVIEW

**Review by kashifest — COMMENTED** [review]
/approve

**Review by smoshiur1237 — DISMISSED** [review]
/lgtm

File: Makefile [hunk 1]
```diff
@@ -20,7 +20,7 @@ SHELL:=/usr/bin/env bash
 
 .DEFAULT_GOAL:=help
 
-GO_VERSION ?= 1.20.4
+GO_VERSION ?= 1.20.8
```
Comment (tuminoid) [inline Makefile:23]: This variable is only used to echo a version on the last line of makefile, so it is not bumping Go.

Go is bumped in Dockerfile.

/hold

File: Dockerfile [hunk 2]
```diff
@@ -13,7 +13,7 @@
 # limitations under the License.
 
 # Support FROM override
-ARG BUILD_IMAGE=docker.io/golang:1.20.7@sha256:741d6f9bcab778441efe05c8e4369d4f8ff56c9a635a97d77f55d8b0ec62f907
+ARG BUILD_IMAGE=docker.io/golang:1.20.8@sha256:1b46d1164aceec47750cc0c3a6f8c44ee8050a0745c07affcef73518544a2b1f
```
Comment (tuminoid) [inline Dockerfile:16]: Where did you get the digest? If I pull golang:1.20.8 I get sha `6b29720c5598cb40b23dd38f522050a698d427c3a5696c5efe1ed22bd45b1396` ? 

Did you pull arm64 on Mac?

**Review by tuminoid — COMMENTED** [review]
/lgtm
/unhold

**Pull request merged**